doc/article/en-US/bundling-your-app-for-deploy.md

```diff
@@ -60,7 +60,7 @@ Now, let's create a `bundle.js` file in `build/tasks/bundle.js` as follows:
           includes: [
             '[*.js]',
             '*.html!text',
-            '*.css!text',        
```

Comment:
This wasn't a bug, but suggested approach to add coma after last array element to avoid unnecessary line change after adding next element in next commit.  eslint and tslint allow you to configure it, because (if i remember correctly) some old IE (that Aurelia doesn't support anyway) had problems with it. So i suggest to not accept this change.